tcf_website/static/course/javascript/user_courses.js

```diff
@@ -0,0 +1,127 @@
+function unsaveCourse(courseID, instructorID, id) {
+    const csrftoken = getCookie("csrftoken");
+
+    $.ajax({
+        type: "POST",
+        headers: { "X-CSRFToken": csrftoken },
+        url: `/unsave_course/${courseID}/${instructorID}`
+    });
+
+    document.getElementById(`course${id}`).remove();
+
+    // if no more courses left
+    if ($(".course-card").length === 0) {
+        document.getElementById("courseToolbar").remove();
+        $("#savedCoursesList").html(`
+            <div class="card col p-5 text-center">
+                <div class="card-body">
+                    <h4 class="card-title">
+                        No Saved Courses <i class="far fa-frown-open fa-fw"></i>
```

Comment:
Maybe there could be a message here directing how to save a course
e.g.
### No Saved Courses
#### Click the `Save` button on a course page to see it here

Reply:
done!